### PR TITLE
[LIMINAL-64] support multiple liminal yaml files

### PR DESCRIPTION
**Pull request by naturalett**
@assapin @aviemzur Thx for your review.

### CODE REVIEW

File: scripts/liminal [hunk 1]
```diff
@@ -129,7 +129,9 @@ def deploy_liminal_apps(path, clean):
     for config_file in config_files:
         click.echo(f"deploying liminal file: {config_file}")
         yml_name = os.path.basename(config_file)
```
Comment (aviemzur) [inline scripts/liminal:131]: `yml_name` is no longer used, this line can be removed

Comment (aviemzur) [reply scripts/liminal:131]: @naturalett

**Review by aviemzur — APPROVED** [review]

File: scripts/liminal [hunk 2]
```diff
@@ -128,8 +128,9 @@ def deploy_liminal_apps(path, clean):
     config_files = files_util.find_config_files(path)
     for config_file in config_files:
         click.echo(f"deploying liminal file: {config_file}")
-        yml_name = os.path.basename(config_file)
-        target_yml_name = os.path.join(environment.get_dags_dir(), yml_name)
+        relative_path = os.path.relpath(config_file)
+        target_yml_name = os.path.join(environment.get_dags_dir(), relative_path)
```
Comment (erickadalberto) [inline scripts/liminal:132]: \blokets

**Review by erickadalberto — COMMENTED** [review]
\

File: scripts/liminal [hunk 3]
```diff
@@ -128,8 +128,9 @@ def deploy_liminal_apps(path, clean):
     config_files = files_util.find_config_files(path)
     for config_file in config_files:
         click.echo(f"deploying liminal file: {config_file}")
-        yml_name = os.path.basename(config_file)
```
Comment (erickadalberto) [inline scripts/liminal:131]: \

File: scripts/liminal [hunk 4]
```diff
@@ -128,8 +128,9 @@ def deploy_liminal_apps(path, clean):
...
-        target_yml_name = os.path.join(environment.get_dags_dir(), yml_name)
```
Comment (erickadalberto) [inline scripts/liminal:132]: \

**Pull request merged**